CHANGELOG

```diff
@@ -1,5 +1,12 @@
 CHANGELOG
 
+2.4.0 - 2015-09-10
```

Comment:
I would set as `UNRELEASED` as we do in PRS because it's hard to know when it will be really released when in the PR